### PR TITLE
Cran hotfix 

### PR DESCRIPTION
**Pull request by DarkShadeKnigh**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ bug, you may also create an associated issue. If possible, try to illustrate you
 *  Please create a Git branch for each pull request (PR). [Click here](https://help.github.com/en/articles/creating-a-pull-request-from-a-fork) for information on how to create a PR.
 *  Your contributed code should roughly follow the tidyverse [style guide](http://style.tidyverse.org). _Exceptions_ from this guide: function names in PascalCase (e.g. NewFunction()) and variable names as camelCase (e.g newVariable <- "hi").
```
Comment (DarkShadeKnigh) [inline CONTRIBUTING.md]: We went back on pascal and now follow tidyverse

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ bug, you may also create an associated issue. If possible, try to illustrate you
...
 *  cchsflow uses [roxygen2](https://cran.r-project.org/package=roxygen2), with
-[Markdown syntax](https://cran.r-project.org/web/packages/roxygen2/vignettes/markdown.html),
+[Markdown syntax](https://bookdown.org/yihui/rmarkdown/markdown-syntax.html),
 for documentation.
 *  cchsflow uses [testthat](https://cran.r-project.org/package=testthat) (TBA). Adding tests to the PR makes it easier for me to merge your PR into the code base.
```
Comment (DarkShadeKnigh) [inline CONTRIBUTING.md]: Implemented now remove (TBA) and "easier for me " seems unprofessional suggest replacing with "makes merging the PR into the code base less prone to bugs"

File: NEWS.md [hunk 3]
```diff
@@ -1,4 +1,23 @@
...
+# cchsflow 1.1.0 (Latest build)
+2020-01-08
+
+## Features
+- Updated `DESCRIPTION` to expand description of package and added The Ottawa
+Hospital as author
+- Changed `data_source` parameter in rec_with_table() to `data`; and changed
+order of parameters to have `variables` after `dataset_name`
+- Included automated testing for functions
+- Refactored functions to account for age below 0 and include warning messages
```
Comment (DarkShadeKnigh) [inline NEWS.md]: name of the exact function that was modified

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# Contributor Code of Conduct
```
Comment (wyusuf068) [inline CODE_OF_CONDUCT.md]: @DougManuel this is the same code of conduct from bllflow. Is there any modifications that need to be done to this?

File: R/label-utils.R [hunk 5]
```diff
@@ -137,6 +173,10 @@ create_label_list_element <- function(variable_rows) {
 #' @return Returns labeled data
 label_data <- function(label_list, data_to_label) {
   for (variable_name in names(label_list)) {
+    if(is.na(label_list[[variable_name]]$type)){
+      warning(paste(variable_name, 'is missing from variable_details or variables(if it was also passed) please verify correct spelling'))
+      next()
```
Comment (DarkShadeKnigh) [inline R/label-utils.R]: add the 80 lines reminder lintr the project

File: R/recode-with-table.R [hunk 6]
```diff
@@ -7,6 +7,28 @@
 #'
 #' @param v1 variable 1
 #' @param v2 variable 2
+#' 
+#' @return boolean value of if v1 and v2 are equal
```
Comment (DarkShadeKnigh) [inline R/recode-with-table.R]: add comma after "of"

File: R/recode-with-table.R [hunk 7]
```diff
@@ -18,7 +40,10 @@ is_equal <- function(v1, v2) {
 }
 
 #' Recode with Table
-#'
+#' 
+#' Recode with Table is responsible for recoding values of a dataset based on
+#' specifications in variable_details.
```
Comment (DarkShadeKnigh) [inline R/recode-with-table.R]: 'based on **the** specifications'

File: README.md [hunk 8]
```diff
@@ -25,7 +25,9 @@ Imagine you want to use BMI (body mass index) for a study that spans CCHS 2001 t
     # load test cchs data - included in cchsflow
     cchs2001test <- cchs2001
     
-    # `variableDetails` is a database in cchsflow that contain the instructions how to transform variables.
+    # `variableDetails` is a database in cchsflow that contain the instructions
```
Comment (DarkShadeKnigh) [inline README.md]: variable_details we switched off from camelCase

File: R/label-utils.R [hunk 9]
```diff
@@ -6,6 +6,40 @@
...
+#'
+#' bmi2012 <- rec_with_table(
+#'   data = cchs2012, variable_details =
+#'     variable_details, dataset_name = "cchs2012", variables = c(
+#'     "HWTGHTM",
+#'     "HWTGWTK", "HWTGBMI_der"
+#'   )
+#' )
+#'
+#' tail(bmi2012)
```
Comment (wyusuf068) [inline R/label-utils.R]: Is it necessary to show head and tails of the transformed data before labelling? Maybe only showing a head of the combined data after applying labels to it.

File: cran-comments.md [hunk 10]
```diff
@@ -1,3 +1,12 @@
+## Resubmission
+
+This is a resubmission. In this version I have:
+* Expanded Descrption section of `DESCRIPTION` to further describe package and
+cite authors of previous packages
+* Added The Ottawa Hospital as author
+* Added executable examples to exported functions
+* Enabled automated testing of functions
```
Comment (DarkShadeKnigh) [inline cran-comments.md]: add thats its through the use of testthat. Also we added some additional features which can be found in the changelog. Perhaps reference the additional features and why we are also adding them and not just the fixes they requested

File: R/label-utils.R (hunk 9 above)
Comment (DarkShadeKnigh) [reply R/label-utils.R]: perhaps replace the head and tails with get_label() before merge after merge then after setting

File: CODE_OF_CONDUCT.md (hunk 4 above)
Comment (DougManuel) [reply CODE_OF_CONDUCT.md]: Not at this time. Looks good.

File: NEWS.md (hunk 3 above)
Comment (DougManuel) [reply NEWS.md]: Suggest that we group items into clearer categories, in order from most important to least important.

## Breaking changes

## New features

## Minor improvements and bug fixes

File: cran-comments.md (hunk 10 above)
Comment (DougManuel) [reply cran-comments.md]: I'd reference the changelog for 'other additional features not requested by CRAN review but arose during revisions'.

**Pull request merged**